src/Steam.js

```diff
@@ -26,8 +26,12 @@ var Steam = function(config, bots) {
 
     this.client.on('error', function(error) {
         console.log((new Date()) + "\tSteam error: " + error);
-        if (!self.user.loggedOn) {
-            self.client.connect();
+        if (!self.client.connected || !self.client.loggedOn) {
+            self._reconnect = setTimeout(function() {
+                console.log((new Date()) + "\tTrying to reconnect");
+                delete self._reconnect;
```

Comment:
I would prefer to change this to `self.reconnect = null` and set the property to null in the constructor.